macros/macros.in

```diff
@@ -234,14 +234,14 @@ package or when debugging this package.\
 %_exec_prefix		%{_prefix}
 %_bindir		%{_exec_prefix}/bin
 %_sbindir		%{_exec_prefix}/sbin
-%_libexecdir		%{_exec_prefix}/libexec
+%_libexecdir		%{_exec_prefix}/lib
 %_datadir		%{_prefix}/share
 %_docdir		%{_datadir}/doc
 %_sysconfdir		/etc
 %_sharedstatedir	%{_prefix}/com
 %_localstatedir		/var
 %_lib			lib
-%_libdir		%{_exec_prefix}/%{_lib}
+%_libdir		%{_exec_prefix}/%{_lib}/%({__dpkg_architecture} -qDEB_HOST_MULTIARCH)
```

Comment:
I'll have to think about this. As it stands, even `debbuild.spec` is "broken" until `%_libdir` and `%_libexecdir` are swapped, and the other specfiles will have to be modified significantly to work on both deb- and rpm-based systems. Plus, the deviation should be documented in `debbuild`'s POD.